tutorials/en/vectara-beginner-app-tutorial-showcase-the-creation-of-vectara-app-in-legal-use-case.mdx

```diff
@@ -0,0 +1,390 @@
+---
+
+Title: "Vectara Beginner App Tutorial: Showcase The Creation Of Vectara App In Legal Use Case"
+Description: "Embark on a step-by-step journey to create an intuitive Vectara application specifically tailored for the legal domain. Uncover the nuances of integrating semantic search and user-friendly interfaces, and get a glimpse into the streamlined process of providing automated legal consultations."
+Author Username: "sanchayt743"
+---
+
+## 1. Introduction to the Vectara Ecosystem: 🌌
+
+Welcome to the realm of Vectara, a platform where search transcends to new horizons powered by Generative AI. Here, I will unravel the essence of the Vectara ecosystem, its fundamental workings, and provide a visual journey through Vectara's official materials to enhance your comprehension.
+
+### 🚀 Overview of Vectara Ecosystem
+
+Vectara is on a mission to redefine how users interact with data and knowledge, facilitating a seamless journey from a user query to the most relevant response. The platform harbors a complete, but easy to customize, search and summarization pipeline, making it an API-based easy-to-use platform to build applications powered by semantic search and Generative AI. Through Vectara, developers are empowered to create GenAI applications with a state-of-the-art retrieval engine and summarization capability, thus elevating the user experience to a realm where questions meet precise answers.
+
+### 🛠 Fundamental Workings and Workflow
+
+The heartbeat of Vectara is its pure neural search platform enriched with production-ready natural language processing. The workflow is simple, yet powerful:
+
+1. **Data Ingestion**: Ingest your data into Vectara's corpus using the Indexing API.
+2. **Query Execution**: Utilize the Search API to run queries against the indexed data, retrieving highly relevant information swiftly.
```

Comment:
Looks much better so far. In this point 2 about query - perhaps I can suggestion that it could say "... retrieving highly relevant information swiftly, and optionally summarizing them into a cohere response to a user query."